nss-day-cohort-51/holiday-road-rocky-mountain-dreamers
# finished css to a working order.  still minor changes needed

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.


# Testing Instructions

Please load the web page and you will see that all of the colors have been added.  Sections have been blocked off accordingly.  Buttons are visible, but non-working at the moment.



# Checklist:

- [X ] I have performed a self-review of my own code
